extension/src/LanguageFunctions.ts

```diff
@@ -1005,18 +1005,27 @@ async function getBaseAppTranslationMap(
     await baseAppTranslationFiles.getBlobs([targetFilename]);
```

Comment:
Shouldn't we take care of the result and do something if result.failed.length > 0?

Right now we're just ignoring errors caused by corrupt files. Sure it'll retry to download the next time user executes function, but the behavior would be a bit magical... In the next line (`localTransFiles = localBaseAppTranslationFiles()`) we assume that all files are downloaded.

I think we should add a couple of tests for getBaseAppTranslationMap(). One of the tests should force `localBaseAppTranslationFiles()` to return also the corrupt json file.

Reply:
Test added for `getBaseAppTranslationMap` :)